schema/server_access_log.schema.json

```
@@ -0,0 +1,248 @@
+{
+    "$id": "https://scality.com/cloudserver/server_access_log.schema.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "title": "Server Access Log",
+    "description": "A server access log",
+    "type": "object",
+    "properties": {
+        "time": {
+            "description": "Epoch timestamp in seconds, recorded when the log record is created.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "hostname": {
+            "description": "Hostname of the worker as returned by os.hostname().",
+            "type": "string"
+        },
+        "pid": {
+            "description": "PID of the worker",
+            "type": "integer",
+            "minimum": 0
+        },
+        "action": {
+            "description": "S3 API action name.",
+            "type": "string"
+        },
+        "accountName": {
+            "description": "Requester account display name.",
+            "type": ["string", "null"]
+        },
+        "accountDisplayName": {
+            "description": "Requester account display name.",
+            "type": ["string", "null"]
+        },
+        "userName": {
+            "description": "Requester IAM display name, or null if the requester is not an IAM user.",
+            "type": ["string", "null"]
+        },
+        "clientPort": {
+            "description": "Requester remote connection port.",
+            "type": "integer",
+            "maximum": 65535,
+            "minimum": 1
+        },
+        "httpMethod": {
+            "description": "Request HTTP method.",
+            "type": "string",
+            "enum": ["GET", "HEAD", "OPTIONS", "TRACE", "PUT", "DELETE", "POST", "PATCH", "CONNECT"]
+        },
+        "bytesDeleted": {
+            "description": "For DeleteObject bytesDeleted is the size in bytes of the deleted object, for DeleteObjects it is the sum of all the deleted objects.",
```

suggestion for a bit less verbosity and more accuracy:

```suggestion
            "description": "For DeleteObject: size in bytes of the deleted object, for DeleteObjects: byte size sum of all the deleted objects",
```

---

done